src/Model/User.php

```diff
@@ -420,9 +421,20 @@ public static function isPasswordExposed($password)
 			'cacheDirectory' => get_temppath() . '/password-exposed-cache/',
 		]);
 
-		$PasswordExposedCHecker = new PasswordExposed\PasswordExposedChecker(null, $cache);
+		try {
+			$passwordExposedChecker = new PasswordExposed\PasswordExposedChecker(null, $cache);
+
+			return $passwordExposedChecker->passwordExposed($password) === PasswordExposed\PasswordStatus::EXPOSED;
+		} catch (\Exception $e) {
```

Comment:
Using `Throwable` since it seems like we're catching technical exceptions too?

Reply:
I'm not sure about that, since the result is a silent success of password exposed. I'd rather only catch userland exceptions at this point. I feel like catching `\Throwable` should be lower in the call chain.